rspace/src/main/scala/coop/rchain/rspace/RSpaceOps.scala

```diff
@@ -285,11 +280,7 @@ abstract class RSpaceOps[F[_]: Concurrent: ContextShift: Log: Metrics: Span, C,
         result <- options match {
                    case None =>
                      for {
-                       _ <- syncF.delay {
```

Comment:
Do we really need `syncF` and `spanF` in the codebase?

Reply:
Yes, they are defined on the trait this abstract class extends.
https://github.com/rchain/rchain/blob/76f14c516bac3d6be2f91318c9b202b4239465bb/rspace/src/main/scala/coop/rchain/rspace/SpaceMatcher.scala#L17-L22

Reply:
I mean, do we even need it in the trait?